main.nf

```diff
@@ -0,0 +1,684 @@
+}
+if(params.email) summary['E-mail Address'] = params.email
+log.info summary.collect { k,v -> "${k.padRight(15)}: $v" }.join("\n")
+log.info "========================================="
+
+
+def create_workflow_summary(summary) {
+
+    def yaml_file = workDir.resolve('workflow_summary_mqc.yaml')
+    yaml_file.text  = """
+    id: 'nf-core-rnafusion-summary'
+    description: " - this information is collected when the pipeline is started."
+    section_name: 'nf-core/rnafusion Workflow Summary'
+    section_href: 'https://github.com/nf-core/rnafusion'
+    plot_type: 'html'
+    data: |
+        <dl class=\"dl-horizontal\">
+${summary.collect { k,v -> "            <dt>$k</dt><dd><samp>${v ?: '<span style=\"color:#999999;\">N/A</a>'}</samp></dd>" }.join("\n")}
+        </dl>
+    """.stripIndent()
+
+   return yaml_file
+}
+
+/*************************************************************
+ * Fusion pipeline
+ ************************************************************/
+
+/*
+ * STAR-Fusion
+ */
+process star_fusion {
+    tag "$name"
+    publishDir "${params.outdir}/tools/StarFusion", mode: 'copy'
+
+    when:
+    params.star_fusion || (params.star_fusion && params.test)
+
+    input:
+    set val(name), file(reads) from read_files_star_fusion
+    file reference from star_fusion_ref.collect()
+
+    output:
+    file '*fusion_predictions.tsv' into star_fusion_fusions
+    file '*' into star_fusion_output
+
+    script:
+    option = params.singleEnd ?: "--right_fq ${reads[1]}"
+    """
+    STAR-Fusion \\
+        --genome_lib_dir ${reference} \\
+        --left_fq ${reads[0]} \\
+        ${option} \\
+        --CPU  ${task.cpus} \\
+        --examine_coding_effect \\
+        --output_dir .
+    """
+}
+
+/*
+ * Fusion Catcher
+ */
+process fusioncatcher {
+    tag "$name"
+    publishDir "${params.outdir}/tools/Fusioncatcher", mode: 'copy'
+
+    when:
+    params.fusioncatcher || (params.fusioncatcher && params.test)
+
+    input:
+    set val(name), file(reads) from read_files_fusioncatcher
+    file data_dir from fusioncatcher_ref.collect()
+
+    output:
+    file 'final-list_candidate-fusion-genes.txt' into fusioncatcher_fusions
+    file '*' into fusioncatcher_output
+
+    script:
+    option = params.singleEnd ? "${reads[0]}" : "${reads[0]},${reads[1]}"
+    """
+    fusioncatcher \\
+        -d ${data_dir} \\
+        -i ${option} \\
+        --threads ${task.cpus} \\
+        -o . \\
+        --skip-blat \\
+        ${params.fc_extra_options}
+    """
+}
+
+/*
+ * Ericscript
+ */
+process ericscript {
+    tag "$name"
+    publishDir "${params.outdir}/tools/Ericscript", mode: 'copy'
+
+    when:
+    params.ericscript && (!params.singleEnd || params.test)
+
+    input:
+    set val(name), file(reads) from read_files_ericscript
+    file reference from ericscript_ref.collect()
+
+    output:
+    file './tmp/fusions.results.total.tsv' into ericscript_fusions
+    file '*' into ericscript_output
```

Comment:
I don't like this so much. All of the nextflow files like `.command.run` etc will be scooped up too. Can it not be made more specific?

Reply:
Fixed